src/odemis/acq/fastem.py

```diff
@@ -415,14 +440,25 @@ def get_abs_stage_movement(self):
         # therefore add half a field in the x-direction and subtract half a field in the y-direction.
         pos_first_tile = (xmin_roa + field_res[0]/2 * px_size[0],
                           ymax_roa - field_res[1]/2 * px_size[1])
+        return pos_first_tile
```

Comment:
Can you add an empty line before the return statement? I find this clearer arranged and matches the rest of the style of this module.